launcher/ui/pages/global/LauncherPage.ui

```diff
@@ -90,16 +87,25 @@
             </property>
            </widget>
           </item>
+          <item row="0" column="1">
+           <widget class="QLineEdit" name="instDirTextBox"/>
+          </item>
+          <item row="3" column="1">
+           <widget class="QLineEdit" name="downloadsDirTextBox"/>
+          </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="iconsDirTextBox"/>
           </item>
-          <item row="2" column="2">
-           <widget class="QToolButton" name="iconsDirBrowseBtn">
+          <item row="3" column="2">
+           <widget class="QToolButton" name="downloadsDirBrowseBtn">
             <property name="text">
-             <string notr="true">...</string>
+             <string>...</string>
```

Comment:
```suggestion
             <string notr="true">...</string>
```

the other `...`s in this page are notr

Reply:
also fixed, seems it got left out in the previous work on this file.